modules/tinymce/src/core/test/ts/browser/init/InitIframeContentAriaLabelTest.ts

```diff
@@ -0,0 +1,33 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks } from '@ephox/mcagar';
+import { Attribute, SugarElement } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.core.init.InitIframeContentAriaLabelTest', () => {
+  const defaultIframeTitle = 'Rich Text Area. Press ALT-0 for help.';
+  const customIframeTitle = 'Cupidatat magna aliquip.';
+  const hookWithDefaultIframeTitle = TinyHooks.bddSetupLight<Editor>({
```

Comment:
As the hooks aren't reused we should be using `McEditor.pFromSettings` and `McEditor.remove` inside each `it` block instead.